README.md

```diff
@@ -0,0 +1,100 @@
+
+<a name="readme-top"></a>
+
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [📝 License](#license)

```

Comment:
- [ ] All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in the repo. The only optional section is FAQ. And I notice that  Contributing / Show your support / Acknowledgments sections are missing. Kindly remedy that detail 🙏🏽